api/TimeRanges.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/TimeRanges",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "6"
```

Comment:
This doesn't add up with the attributes that return `TimeRanges`, but commented on https://github.com/mdn/browser-compat-data/issues/7843#issuecomment-747482568 instead of asking for that to be fixed in this PR.